cmd/kubeadm/app/cmd/phases/init/waitcontrolplane.go

```diff
@@ -47,11 +48,18 @@ var (
 		- 'journalctl -xeu kubelet'
 
 	Additionally, a control plane component may have crashed or exited when started by the container runtime.
-	To troubleshoot, list all containers using your preferred container runtimes CLI, e.g. docker.
+	To troubleshoot, list all containers using your preferred container runtimes CLI.
```

Comment:
Maybe we should add other runtimes as well.

Reply:
I was looking to add containerd as well, but they dont have a full fledged CLI currently that we can translate the help commands too. Seems that you can use `crioctl` to manage it so we could lump together cri-o and containerd and print the crio output. As it includes the socket config, it should work for both.

No idea about any other runtimes currently, other that the big three (docker, cri-o, containerd)

Reply:
Nice!

Reply:
why remove docker from this sentence - it's an indication for newbie users that docker is a "container runtime"?